src/configurator/index.js

```diff
@@ -20,6 +20,7 @@ const defaultOptionsCLIonly = {
   production: false,
   static: false,
   server: false,
+  open: false,
```

Comment:
Alright, so this should be disabled via package.json. but can also be enabled via cli